### PR TITLE
Accordion

### PR DESCRIPTION
**Pull request by kevin-quach**
https://trello.com/c/moDWmzBk

**Changes:**

- [x] Added Accordion Component
  

**How to test:**

- Create an accordion based on the spec outlined
- Accordion opens as specified
  

**Todo:**
- [x] Accordion options
- [x] Animation
- [x] Improved default typography
- [x] +1

### CODE REVIEW

File: public/js/accordion.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+$(document).ready(function() {
```
Comment (Magitron) [inline public/js/accordion.js]: We should probably pick and agree on one way of writing this `$(document).ready(function()` or `(function($)`. Can you look up the current best practice, change it to the best practice method, and also add it to the js code standards in the wiki? 🤓

Comment (Magitron) [reply public/js/accordion.js]: And then please just make a quick card so we can remember to change the rest of the js we currently have later

File: public/sass/components/_accordion.scss [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+.accordion-body {
+  display: block;
+  overflow: hidden;
+  padding: $gutter*1.5;
+  border-bottom: 1px solid $concrete;
+  font-size: 0.86em;
+  line-height: 1.5em;
+}
```
Comment (Magitron) [inline public/sass/components/_accordion.scss]: image

File: views/accordion.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+			</div>
+		</li>
+		<li>
+			<div class="accordion-header">Item 4</div>
+			<div class="accordion-body">
+				Lorem ipsum dolor sit amet, consectetur adipisicing elit. Labore voluptas ipsam maiores animi? Accusantium, hic? Sed totam quis, ipsum, laudantium magnam reiciendis quidem mollitia. Sapiente quam, iusto provident ipsa nulla.Lorem ipsum dolor sit amet, consectetur adipisicing elit. Labore voluptas ipsam maiores animi? Accusantium, hic? Sed totam quis, ipsum, laudantium magnam reiciendis quidem mollitia. Sapiente quam, iusto provident ipsa nulla.Lorem ipsum dolor sit amet, consectetur adipisicing elit. Labore voluptas ipsam maiores animi? Accusantium, hic? Sed totam quis, ipsum, laudantium magnam reiciendis quidem mollitia. Sapiente quam, iusto provident ipsa nulla.Lorem ipsum dolor sit amet, consectetur adipisicing elit. Labore voluptas ipsam maiores animi? Accusantium, hic? Sed totam quis, ipsum, laudantium magnam reiciendis quidem mollitia. Sapiente quam, iusto provident ipsa nulla.
+			</div>
+		</li>
+    </ul>
```
Comment (Magitron) [inline views/accordion.html]: I know it looks weird here, but there's just one indentation for the `</ul>` that needs to be in one more. Just a quick fix while you make the other changes.

File: public/sass/components/_accordion.scss (hunk 2 above)
Comment (Magitron) [reply public/sass/components/_accordion.scss]: Looks awesome! 

I just noticed that the CSS Comb doesn't seem to be working for you. Let's take a look at that when you have a second.

File: public/js/main.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+
+$( document ).ready(function() {
+
+  $('[data-toggle]').click(function() {
+    var target = $(this).attr('data-toggle');
+    $(target).toggleClass('active');
+  });
+
+});
+
```
Comment (Magitron) [inline public/js/main.js]: Are we using this? I think the way you set it up above looks good. If this worked for you though, you could save this code and reimplement it for the modal or drop down or tooltip or something similar.

File: public/js/accordion.js (hunk 1 above)
Comment (kevin-quach) [reply public/js/accordion.js]: Document.ready removed from accordion.js, will use `(function($)` moving forward! :D 

Will document this. For learning purposes, they are the same thing with the latter being a shortcut and better prevents library conflicts.

**Pull request merged**